test-e2e/tests/live_channel_test.ts

```diff
@@ -0,0 +1,237 @@
+import assert from 'assert';
+
+import constants from '../utils/constants';
+
+const selectedProgramBackgroundColor = 'rgb(204, 204, 204)';
+const liveProgramOutline = 'rgb(255, 255, 255) solid 2px';
```

Comment:
Name convention switch

```suggestion
const programSelectedBackgroundColor = 'rgb(204, 204, 204)';
const programLiveOutline = 'rgb(255, 255, 255) solid 2px';
```